dusty/commands/setup.py

```diff
@@ -64,6 +65,21 @@ def _get_and_configure_nginx_includes_dir():
     _get_raw_input('\n'.join(textwrap.wrap('You have a custom nginx config setup. Could not find an nginx.conf file. Please read our docs to see what is needed for the nginx config.  Once you have figured it out, please use `dusty config` command to adjust your `nginx_includes_dir`', 80)))
     return ''
 
+def _get_boot2docker_vm_size():
+    memory_megs = virtual_memory().total / 2**20
+    sixteen_megs = 16 * 2**10
```

Comment:
I changed my mind.  This is a bad point.  A megabyte is 2 ^ 20.   So what sixteen_megs here is the total bytes for 16 megs.